public/usage-examples/graphics/fill_rectangle_on_window/fill_rectangle_on_window-1-simple-oop.cs

```diff
@@ -0,0 +1,19 @@
+using SplashKitSDK;
+
+namespace Program
+{
+  public class Program
+  {
+    public static void Main()
+    {
+      Window myWindow = SplashKit.OpenWindow("Fill Rectangle On Window", 800, 600);
+
+      // Fill rectangle on the window and refresh
+      SplashKit.FillRectangleOnWindow(myWindow, SplashKit.ColorBlue(), 300, 250, 200, 100);
```

Comment:
This can be made more OO by calling the function from the object ie `myWindow.FillRectangle` 
similarly with the color, instead of `SplashKit.ColorBlue()` try `Color.Blue`